apt-upgrades/apt-upgrades

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Copyright (C) 2015 James Murphy                                                  
+# Licensed under the terms of the GNU GPL v2 only.                                 
+#                                                                                  
+# i3blocks blocklet script to display pending system upgrades 
+
+# FontAwesome refresh symbol, change if you do not want to install FontAwesome
+REFRESH_SYMBOL="\uf021 "
+
+# By default, show both the symbol and the numbers
+SYMBOL_ONLY=0
+
+# Colors for when there is/isn't a pending upgrade
+REFRESH_COLOR="#00FF00"
+NONREFRESH_COLOR="#FFFFFF"
+
+while getopts s:oc:n:h opt; do
+    case "$opt" in
+        s) REFRESH_SYMBOL="$OPTARG" ;;
+        o) SYMBOL_ONLY=1 ;;
+        c) REFRESH_COLOR="$OPTARG" ;;
+        n) NONREFRESH_COLOR="$OPTARG" ;;
+        h) printf "Usage: apt-upgrades [-s refresh_symbol] [-o] [-c refresh_color] [-n nonrefresh_color] [-h]\nOptions:\n-s\trefresh symbol\n-o\tshow symbol only, no numbers\n-c\tcolor when upgrade pending\n-n\tcolor when no upgrade pending\n-h\tshow this help text\n" && exit 0;;
+    esac
+done
+
+read upgraded new removed held < <(
+aptitude full-upgrade --simulate --assume-yes |\
+    grep -m1 '^[0-9]\+ packages upgraded,' |\
+    tr -cd '[0-9 ]' |\
```

Comment:
> [SC2021](https://github.com/koalaman/shellcheck/blob/0f00de80fd1dd16c21b603625d90fa92795ca22f/ShellCheck/Analytics.hs#L1640) Don't use [] around ranges in tr, it replaces literal square brackets.

I suggest passing your shell scripts in [Shellcheck](http://www.shellcheck.net/) which is an amazing tool to statically analyse common pitfalls. Indeed, if you put `[]` in `tr`, they're not removed:
```shell
$ echo 'foo 73 [bar 112]' | tr -cd '[0-9 ]'
 73 [ 112]
```